app/models/region.rb

```diff
@@ -168,6 +168,11 @@ def appointed_patients
     Patient.joins(:appointments).where(appointments: {facility: facility_regions.pluck(:source_id)})
   end
 
+  def diabetes_management_enabled?
+    Flipper.enabled?(:diabetes_management_reports) &&
+      facilities.where(enable_diabetes_management: true).exists?
+  end
```

Comment:
[rename] Lets call this `diabetes_reports_enabled?`.

Reply:
This is a method on the region, which checks if any of its facilities have diabetes management enabled. The feature flag check here is just for convenience. When we remove the feature flag, the name should make more sense.

Reply:
I'd suggest breaking it like so in that case, since someone might want to use `diabetes_management_enabled` in a different context
```
def diabetes_management_enabled?
   facilities.where(enable_diabetes_management: true).exists?
end

def diabetes_reports_enabled?
   Flipper.enabled? && diabetes_management_enabled?
```